### ISSUE TITLE
PT012: False positive on context manager with statement

### ISSUE THREAD

**Issue by jdufresne** [m-burst/flake8-pytest-style#93]
Input:

```py
def my_test():
    with pytest.raises(MyError):
        with my_context_manager_under_test():
            pass
```

Expected: No error. This is the most minimal way to express the test without additional side effects. The test must enter the context manager to be effective.

Actual: PT012 pytest.raises() block should contain a single simple statement

**Comment by m-burst** [comment]
Hi @jdufresne,
I'll see what I can do to fix this. In the meantime, you can use the following workarounds:
1. extract the `with my_context_manager_under_test(): pass` block into a separate function
2. explicitly call `my_context_manager_under_test().__enter__()`
3. add a noqa comment

I would also like to note that your test in its current form may pass if the exception is raised during `__enter__` or during `__exit__`. If you need to test that the exception is raised exactly during `__enter__`, it might be better to write this test in some other way.

**Comment by m-burst** [comment]
Hi @jdufresne,
This is fixed in v1.4.0 which I've just released.
Empty `with` statements (containing only a single `pass`) are now considered a simple statement in the PT012 check.

**Comment by jdufresne** [comment]
Thanks!
